File: Sakila_Net/Sakila_Net/VB_SakilaLib/DAO/DAOLanguage.vb
'using System;
Imports  System.Data
'using System.Collections.Generic;
'using System.Configuration;
'using System.Collections;
Imports  SakilaLib.Cl.Harrys.Sakila.Model
'using Cl.Harrys.BD;

Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace DAO

                Public Class DAOLanguage
                    'Inherits SakilaLib.Cl.Harrys.Sakila.DAOBD.DAOLanguage
                    Public Shared  Function SelectAll(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAOBD.DAOLanguage.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public Shared  Function SelectOne( language as ClLanguage) as Boolean
                        Return DAOBD.DAOLanguage.SelectOne(language)     
                    End Function
                    Public Shared  Function SqlInsert( language as ClLanguage) as Integer
                        Return DAOBD.DAOLanguage.SqlInsert(language)     
                    End Function
                    Public Shared  Function SqlDelete( language as ClLanguage) as Integer
                        Return DAOBD.DAOLanguage.SqlDelete(language)     
                    End Function
                    Public Shared  Function SqlUpdate( language as ClLanguage) as Integer
                        Return DAOBD.DAOLanguage.SqlUpdate(language)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  

File: Sakila_Net/Sakila_Net/CS_SakilaLib/DAO/DAOLanguage.cs
//using System;
using System.Data;
//using System.Collections.Generic;
//using System.Configuration;
//using System.Collections;
using Cl.Harrys.Sakila.Model;
namespace  Cl.Harrys.Sakila.DAO
{   
  public class DAOLanguage 
  {
       public static DataTable sqlSelectAll(int nPagina,int cantidad)
	   {
           return DAOBD.DAOLanguage.selectAll(nPagina,cantidad);
	   }
       public static bool sqlSelect(ClLanguage language)
	   {
            return DAOBD.DAOLanguage.SelectOne(language); 
	   }
	   public static int sqlInsert(ClLanguage language)
	   {
            return DAOBD.DAOLanguage.sqlInsert(language);
       }
	   public static int sqlDelete(ClLanguage language)
	   {
            return DAOBD.DAOLanguage.sqlDelete(language);
       }
	   public static int sqlUpdate(ClLanguage language)
	   {
            return DAOBD.DAOLanguage.sqlUpdate(language);
       }
   }
}
